Arm/rover_arm_system.hpp

```diff
@@ -67,8 +67,9 @@ class RoverArmSystem : public sjsu::common::RoverSystem
   RoverArmSystem(sjsu::arm::Joint & rotunda,
                  sjsu::arm::Joint & shoulder,
                  sjsu::arm::Joint & elbow,
+                 sjsu::arm::Hand & hand,
                  sjsu::arm::WristJoint & wrist)
-      : rotunda_(rotunda), shoulder_(shoulder), elbow_(elbow), wrist_(wrist)
+      : rotunda_(rotunda), shoulder_(shoulder), elbow_(elbow), hand_(hand), wrist_(wrist)
```

Comment:
This is why main.cpp has an error, it needs to reflect the new constructor which now has a hand argument